Source/Core/VideoCommon/GraphicsModSystem/Config/GraphicsMod.cpp

```
@@ -112,6 +112,47 @@ std::string GraphicsModConfig::GetAbsolutePath() const
   }
 }
 
+void GraphicsModConfig::SerializeToConfig(picojson::object* obj) const
+{
+  if (!obj) [[unlikely]]
```

While we have the rule in our style guide (that I put in there and want to slap myself for) where non-mutable refs should be pointers instead. Maybe we should subvert that here and turn these unlikely's into "never at all"s (pragmatically, anyway), especially when the alternative is silently doing nothing.

---

Thanks lioncash.  I definitely get bit by that rule quite a bit (all over graphics mod stuff).  I updated this code to use references instead.